### DISCUSSION TITLE
Using PyG and C++

### DISCUSSION THREAD

**Discussion by anthonysirico** [pyg-team/pytorch_geometric#6189] [Q&A]
Are there any examples of using PyG and C++?

**Comment by rusty1s** [comment]
This might be a good starting point: https://github.com/pyg-team/pytorch_geometric/tree/master/examples/cpp

**Comment by Ahmed-Rokenddine** [comment]
this might come usefull https://codescracker.com/cpp/program/cpp-programming-examples.htm
